### PR TITLE
[plugin.video.rivedila7] 2.0.0

### PR DESCRIPTION
**Pull request by luivit**
### Description
bugfix
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] 

Additional information :
bump to 2.0.0 version

### CODE REVIEW

File: plugin.video.rivedila7/addon.py [hunk 1]
```diff
@@ -8,20 +8,22 @@
 def enabled_addon(addon):
     data = json.loads(xbmc.executeJSONRPC('{"jsonrpc":"2.0","method":"Addons.GetAddonDetails","params":{"addonid":"'+addon+'","properties":["enabled","installed"]},"id":5}'))
     if "result" in data:
-        xbmc.log("Addon is installed. Enabling if disabled.")
-        if not data["result"]["addon"]["enabled"]:
+        xbmc.log('Add-on InputStream Adaptive installed',xbmc.LOGNOTICE)
```
Comment (enen92) [inline plugin.video.rivedila7/addon.py]: please use logdebug

**Comment by enen92** [comment]
Also, the news item in addon.xml is only for the latest version of the addon. If you need to keep a full changelog use changelog.txt although it won't be visible from the kodi interface.

**Comment by Testato** [comment]
@enen92 
please may you explain why the inputstream dependency fail ?
I tested the plugin on kodi17 and kodi18 without problem

**Comment by Rechi** [comment]
You are requiring inputstream.adaptive as non optional dependency. The submission is for Jarvis (v16) and higher, but inputstream.adaptive isn't available for Jarvis.

**Pull request closed without merging**